src/modules/cmdpal/extensionsdk/nuget/Microsoft.Windows.CommandPalette.Extensions.nuspec

```diff
@@ -1,36 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
(More stuff that I think @ericjohnson327 will probably end up resurrecting soon-ish)